contracts/router/SubaccountGelatoRelayRouter.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.0;
+
+import "../order/IBaseOrderUtils.sol";
+import "../router/Router.sol";
+import "../feature/FeatureUtils.sol";
+import "../subaccount/SubaccountUtils.sol";
+import "./BaseGelatoRelayRouter.sol";
+
+contract SubaccountGelatoRelayRouter is BaseGelatoRelayRouter {
+    struct SubaccountApproval {
+        address subaccount;
+        uint256 expiresAt;
+        uint256 maxAllowedCount;
+        bytes32 actionType;
+        uint256 deadline;
+        uint256 nonce; // for replay attack protection
+        bytes signature;
+    }
+
+    bytes32 public constant _SUBACCOUNT_APPROVAL_TYPEHASH =
```

Comment:
just curious, why is there an underscore for this?

Reply:
no specific reason, have removed